.circleci/config.yml

```diff
@@ -1,189 +1,151 @@
+# CircleCI 2.0 configuration file. See <https://circleci.com/docs/2.0/language-python/>.
 version: 2
-
 jobs:
-  checkout:
-    working_directory: ~/openfisca-core
+  build_python2:
     docker:
       - image: python:2.7.14
 
     steps:
       - checkout
 
+      - restore_cache:
+          key: v1-py2-{{ checksum "setup.py" }}
+
       - run:
-          name: Fetch remote refs
-          command: git fetch
+          name: Create a virtualenv
+          command: |
+            mkdir -p /tmp/venv/openfisca_core
+            virtualenv /tmp/venv/openfisca_core
+            echo "source /tmp/venv/openfisca_core/bin/activate" >> $BASH_ENV
+
+      - run:
+          name: Install dependencies
+          command: |
+            pip install --upgrade pip twine wheel
+            pip install --editable .[test] --upgrade
+          #  pip install --editable git+https://github.com/openfisca/country-template.git@BRANCH_NAME#egg=OpenFisca-Country-Template  # use a specific branch of OpenFisca-Country-Template
```

Comment:
Should be Core.

Reply:
OpenFisca Country Template is a test dependency of Core. This line allows you to run the tests with a specific branch of the country template.

This is useful when working and breaking changes that imply editions both on core and country template.

Reply:
Oh yeah good point!